packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_screenshot.spec.ts

```diff
@@ -1,262 +1,266 @@
 // @ts-ignore / session support is needed for visiting about:blank between tests
 context('multi-domain screenshot', { experimentalSessionSupport: true }, () => {
-  beforeEach(() => {
```

Comment:
The test changes in here are related to  not being able to call cy.viewport between a click that switches domains in the beforeEach and the switchToDomain command in the test.